core/templates/pages/teach-page/teach-page.scripts.ts

```diff
@@ -22,4 +22,4 @@ require('pages/teach-page/teach-page.module.ts');
 require('App.ts');
 require('base-components/oppia-root.directive.ts');
 
-require('pages/teach-page/teach-page.component.ts');
+require('base-components/base-content.directive.ts');
```

Comment:
Why is this added here? (It was not needed before)

Reply:
Actually, this was giving issues on develop as well! The base components were loading extremely late (topbar, logo etc), so I realized it was probably a base component issue. I saw it was already present in few other pages, so I added it here

Reply:
Is it working fine now -- I mean faster?

Reply:
@bansalnitish, do you remember I was adding this line in each module (https://github.com/oppia/oppia/pull/9576#discussion_r443145447). I actually ran into errors, but I think there is something fishy about this base-directive.

Reply:
Yes!

Reply:
Yup, I remember Srijan.